binary.h

```diff
@@ -0,0 +1,91 @@
+/*
+  Copyright (C) 2013 - 2014 CurlyMo
+
+  This file is part of pilight.
+
+  pilight is free software: you can redistribute it and/or modify it under the
```

Comment:
Not sure we can add a file protected by GPL to an LGPL project :(

Reply:
i think we can. We just use it in the open project without any modifications.